azerothcore/azerothcore-wotlk
# fix(DB/Items): Unhatched Jubling Egg obtaining + RP

## Changes Proposed:

This PR proposes changes to:

Egg timer is handled via category cooldown

## Issues Addressed:

- Progress https://github.com/azerothcore/azerothcore-wotlk/issues/6736
- Progress https://github.com/chromiecraft/chromiecraft/issues/1091

## SOURCE:

The changes have been validated through:

https://www.youtube.com/watch?v=EhKYWDTKjQE
https://www.wowhead.com/wotlk/quest=7946/spawn-of-jubjub

## Tests Performed:

This PR has been:


## How to Test the Changes:



.event start 5
.go c 137684
.additem 11325 2

Notice no quest
Notice gossip (notice it again after quest completion)
use the item within 5 yards of Morja
should talk (RP is partially missing)
Quest should appear
Try to spam the item, you can't
Should reset after 1 min and hide quest
Complete the quest and try to open the egg, you can't 

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
